docs/security/customize-user-lockout.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Customize max login attempts and duration of user lockout
+description: Use Clerk to limit the number of times a user can attempt to log in before they are locked out of their account and how long such lockouts last.
+---
+
+# Customize the max attempts before getting locked out and duration of lock out
+
+<Callout type="info">
+This feature is applicable to user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) or [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
+</Callout>
+
+1. In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **User & Authentication > [Attack Protection](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection)**.
+2. To change the number of failed attempts before a user is locked out, under **Maximum attempts,** select "Custom attempts" and enter a new number of failed attempts allowed. (The default is 100 attempts.)
+3. To change the duration, under **Duration,** select "Custom duration" and select the unit of time (minutes/hours/days/years) and enter the number of units you want lockouts to last.
+4. Select "Apply changes" to apply your settings.
+
+<Images
+width={1512}
+height={914}
+src="/docs/images/security/userlock_custom-attempts-and-duration.png"
+alt=" "
+/>
```

Comment:
The text doesn't seem to follow the screenshot.

In the screenshot, there is a **Maximum attempt limit** section, and I don't see "Custom attempts" anywhere.
Following the screenshot, I think you meant **Lockout duration** rather than **Duration**. But the text says "select "Custom duration"" and I also don't see that in the screenshot either. 

Is this the correct screenshot? Or does the text need to be updated.

Also, "Apply changes" can  be bolded to **Apply changes**